meinberlin/fixtures/votes.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "model": "meinberlin_votes.tokensalt",
+    "pk": 1,
+    "fields": {
+      "salt": "OlF1DYwSFQFxPZX9",
+      "module": 10
+    }
+  },
+  {
+    "model": "meinberlin_votes.tokensalt",
+    "pk": 2,
+    "fields": {
+      "salt": "Yg3arzV0I35RVa2o",
+      "module": 2
+    }
+  },
+  {
+    "model": "meinberlin_votes.tokensalt",
+    "pk": 3,
+    "fields": {
+      "salt": "aUotLnFYnNsZwNia",
+      "module": 11
+    }
+  },
+  {
+    "model": "meinberlin_votes.votingtoken",
+    "pk": 1,
+    "fields": {
+      "token": "votingtoken1",
```

Comment:
do we document things like this somewhere?

Reply:
I'm not sure, @Rineee @fuzzylogic2000 ?

Reply:
I think not yet ..